### PR TITLE
Peer programming with Christy Erdem and Andrew on Christy's birthday

### PR DESCRIPTION
**Pull request by christylaguardia**
#### Single-line Summary
**Today, Christy, Erdem and Andrew paired together. It took about 3 hours**

#### Reflect and summarize on your process for each `TODO` item :  
  1. First, we started on scripts on the index.html
  2. Next, we added the data-author attribute on article.js
  3. Then, we worked on the filters in articleView.js
  4. Then, we worked on the Read on / Show less functionality

#### Checklist (before submitting, fill in each set of square brackets with an 'x')
- [x] We have titled the Pull Request similar to our branch name (ex: 'brian-rick'). 
- [x] This PR includes commits from both myself and my partner; e.g. We followed good pair programming practices by switching driver/navigator roles.
- [x] There is no extraneous, unrelated code included in this PR.
- [x] We have summarized our `TODO:` process above.

### CODE REVIEW

File: starter-code/index.html [hunk 1]
```diff
@@ -54,7 +54,7 @@
             </div>
           </header>
           <section class="article-body"></section>
-          <a href="#" class="read-on">Read on &rarr;</a>
+          <a href="#" class="read-on">Read on</a>
```
Comment (JPGothro) [inline starter-code/index.html]: Why did you remove the arrow?

Comment (christylaguardia) [inline starter-code/index.html]: I think we couldn't get the arrow to show up when we were changing it to 'Show less' and then back to 'Read on'

**Comment by JPGothro** [comment]
An arrow going the other way would be "&larr".

On Apr 26, 2017 9:10 PM, "Christy La Guardia" <notifications@github.com>
wrote:

*@christylaguardia* commented on this pull request.
------------------------------

In starter-code/index.html
<https://github.com/cfpdx-301night-spring-2017/03-jQuery-and-events/pull/1#discussion_r113611908>
:

> @@ -54,7 +54,7 @@
             </div>
           </header>
           <section class="article-body"></section>
-          <a href="#" class="read-on">Read on &rarr;</a>
+          <a href="#" class="read-on">Read on</a>

I think we couldn't get the arrow to show up when we were changing it to
'Show less' and then back to 'Read on'

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/cfpdx-301night-spring-2017/03-jQuery-and-events/pull/1#discussion_r113611908>,
or mute the thread

File: starter-code/scripts/articleView.js [hunk 2]
```diff
@@ -60,6 +84,20 @@ articleView.setTeasers = function() {
 
     // STRETCH GOAl!: change the 'Read On' link to 'Show Less'
   */
-};
+  $('.read-on').click(function() {
+    if($(this).text() === 'Read on') {
+      $(this).siblings('section').find('p').fadeIn(423);
+      $(this).text('Show Less');
```
Comment (JPGothro) [inline starter-code/scripts/articleView.js]: Did you try setting it this way?:   $(this).html('Show Less &larr;');

**Comment by kbrumer** [comment]
Very good!
